machines/Makefile

```diff
@@ -1,50 +1,44 @@
-find-dir-with = $(shell /usr/bin/perl -e 'chomp($$_ = `pwd`); while ($$_ ne "" && ! -e "$$_/$(1)") { m:(.*)/[^/]+/??:; $$_ = $$1; } print;')
-
-ifndef BSG_MANYCORE_DIR
-  export BSG_MANYCORE_DIR := $(call find-dir-with,.BSG_MANYCORE_ROOT)
-endif
+.DEFAULT_GOAL = all
 
+export BSG_MANYCORE_DIR := $(shell git rev-parse --show-toplevel)
 export BASEJUMP_STL_DIR := $(BSG_MANYCORE_DIR)/../basejump_stl
 
 # Sets "VCS" variable
 include $(BSG_MANYCORE_DIR)/../bsg_cadenv/cadenv.mk
 
-MACHINES = 9x15 4x4
-
-SIMV = $(addsuffix /simv, $(MACHINES))
-SIMV_DEBUG = $(addsuffix /simv-debug, $(MACHINES))
-SIMV_DAIDIR = $(addsuffix .daidir, $(SIMV) $(SIMV_DEBUG))
+# Include source lists
+include Makefile.include
```

Comment:
why do we need to create another file? can we just have those lines include in this Makefile?

Reply:
It's a personal choice. I wanted to separate any "listings" we do in Makefile.include and primarily have rules in Makefile.

Reply:
okay, I think if we are not including Makefile.include in some other places, there is no reason to have this in a separate file. It creates more work for the reader, that now the reader needs to open this file to understand what's going on. It's kind of the same reason why you are making this refactor. People don't like additional steps.

Reply:
Yes, include file exclusively is for file listings and can be easily included in other places when the need arises.

Reply:
Sure, but let's not create this file until that need actually arises.